libs/image/include/image/ImageSampler.h

```diff
@@ -20,6 +20,8 @@
 #include <image/LinearImage.h>
 
 #include <utils/compiler.h>
+#include <memory>
+#include <cstdint>
```

Comment:
we don't expose the STL in public headers. use `stdint.h` instead and don't use `memory` in public APIs.